src/Playroom/Snippets/Snippets.css.ts

```diff
@@ -48,6 +52,8 @@ export const snippet = style([
     paddingX: 'xlarge',
   }),
   {
+    marginInline: snippetsBorderSpace,
```

Comment:
Setting the margin on the individual snippets rather than the container to avoid margin being applied before the actual scrollbar. This is an issue in master